scripts/gulp/src/tasks/browserAdapters.ts

```diff
@@ -17,39 +16,24 @@ export type Browser = {
 };
 
 export async function createChrome(): Promise<Browser> {
-  const options = safeLaunchOptions();
-  let browser: puppeteer.Browser | undefined;
-  let attempt = 1;
-  while (!browser) {
-    try {
-      browser = await puppeteer.launch(options);
-    } catch (err) {
-      if (attempt === 5) {
-        console.error(`Puppeteer failed to launch after 5 attempts`);
-        throw err;
-      }
-      console.warn('Puppeteer failed to launch (will retry):');
-      console.warn(err);
-      attempt++;
-    }
-  }
+  const browser = await launch();
 
   console.log(`Chromium version: ${await browser.version()}`);
 
   return {
     openPage: async url => {
-      const page = await (browser as puppeteer.Browser).newPage();
+      const page = await browser.newPage();
 
       await page.goto(url);
 
       return {
         executeJavaScript: async code => {
           return page.evaluate(code);
         },
-        close: async () => page.close(),
+        close: () => page.close(),
```

Comment:
unnecessary async, those `close` calls already return `Promise`, while `await` will unwrap those it is not a good practice to create promise over promise for no reason